src/cljs/athens/db.cljs

```diff
@@ -47,8 +47,8 @@
        (map #(map edn/read-string %))
        (map #(cons :db/add %))))
 
-(defn json-str-to-edn
-  "Convert a JSON str to EDN. May receive JSON through an HTTP request or file upload."
+(defn json-str-to-vector
+  "Convert a JSON str to a clojure vector. May receive JSON through an HTTP request or file upload."
```

Comment:
I don't think this is correct? It depends on the input if it's a vector. Can you explain why you changed this?

Reply:
Agree @jeroenvandijk. I think `edn` is a more descriptive while still generic name than `vector`.

Reply:
Yes, I might have changed this back and forth sometimes rsrsrs not clear if was valuable or not have this argument here. Anyway, the function signature for [js->clj](https://cljs.github.io/api/cljs.core/js-GTclj) just states the conversion from js array to clojurescript vector and to be honest I was expecting the output of ":block/order" to be a keyword instead of a string from this function when I read `json-str-to-edn`. 

I don't mind changing it back again.